xep-0157.xml

```diff
@@ -192,6 +201,15 @@
       var='security-addresses'
       type='list-multi'
       label='One or more addresses for communication related to security concerns'/>
+  <field
+      var='status-addresses'
+      type='list-multi'
+      label='One or more addresses for service status'>
+    <validate xmlns='http://jabber.org/protocol/xdata-validate'
```

Comment:
@Flowdalic What is the source for being able to use validation stuff in XEP-0068 form type definitions, the schema of which is, AFAIK, normatively defined in XEP-0068 and does not include any validation stuff?

Reply:
It also does not disallow additional elements.

Reply:
Do you have a source which makes that argument count for registries?

Reply:
No.

But there is also no source for extendability of extension elements in XMPP (I am aware of). Yet it is common practice.

This is in line with this spirit of extensibility of XMPP. It mimics how data form validation extend form fields on the wire protocol on the registry layer. 

I think it is very desirable to have that information there. But if you want to start a generic discussion if this the right way, which I think it clearly is, or if this belongs at all in the registry, which I think it clearly does, then we should continue this discussion on standards@.